src/content/developers/docs/consensus-mechanisms/index.md

```diff
@@ -16,9 +16,9 @@ To better understand this page, we recommend you first read our [introduction to
 
 ## What is consensus? {#what-is-consensus}
 
-By consensus, we mean that a general agreement has been reached. Consider a group of five people going to the cinema. If three out of five agree on a film, a consensus is achieved — majority rules.
+By consensus, we mean that a general agreement has been reached. Consider a group of few people going to the cinema. If there are no principal disagreements on a film, then a consensus is achieved. Participants either get the group on their side, either side with the group. In the extreme case the group will eventually split.
```

Comment:
Choice of word improvements: 

```suggestion
By consensus, we mean that a general agreement has been reached. Consider a group of people going to the cinema. If there is not a disagreement on a proposed choice of film, then a consensus is achieved. Participants either get the group on their side, either side with the group. In the extreme case the group will eventually split.
```

Reply:
@elshigori What is this supposed to say: "Participants either get the group on their side, either side with the group."

Reply:
It's just an outline of options for any consensus to move forward: 
* propagate your position among peers, 
* adjust your position to others, 
* or split.